general-configuration-guide/customize-ui.md

```diff
@@ -34,9 +34,7 @@ tinymce.init({
 });
 ```
 
-Creating skins for {{site.productname}} is easy with the {{site.productname}} Skin tool project. See [Creating a skin]({{ site.baseurl }}/advanced/creating-a-skin/) for more details.
-
-> Note: The  [{{site.productname}} Skin Creator](http://skin.tiny.cloud/) only supports TinyMCE 4.
+To create a skin interactively, visit [the {{site.productname}} 5 Skin Tool](http://skin.tiny.cloud/t5/). For information on manually creating skins for {{site.productname}}, see [Creating a skin]({{ site.baseurl }}/advanced/creating-a-skin/).
```

Comment:
`{{site.productmajorversion}}` instead of `5`?

Reply:
leaving 5 hardcoded just in case we don't update it in time for TinyMCE 6 release (whenever that is)